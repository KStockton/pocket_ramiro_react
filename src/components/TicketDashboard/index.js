import React, { Component } from 'react';
import CardContainer from '../CardContainer';


export class TicketDashboard extends Component {
  
  
  render() {
    return (
      <main>
      <CardContainer/>
    </main>
  )
};
}

export default TicketDashboard;