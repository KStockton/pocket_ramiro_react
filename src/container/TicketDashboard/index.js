import React, { Component } from 'react';
import { connect } from 'react-redux';
import { getActiveTickets } from '../../thunks/getActiveTickets';

export class TicketDashboard extends Component {

  componentDidMount() {
    this.props.getActiveTickets();
  }

  render() {

    return (
      <main>
      </main>
    )
  }
}

const mapStateToProps = (state) => ({
  tickets: state.tickets
});

const mapDispatchToProps = (dispatch) => ({
  getActiveTickets: () => dispatch(getActiveTickets())
});

export default connect(mapStateToProps, mapDispatchToProps)(TicketDashboard);