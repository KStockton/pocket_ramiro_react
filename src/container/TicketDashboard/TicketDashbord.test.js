import React from 'react';
import { shallow } from 'enzyme'
import TicketDashboard from './index';

describe('TicketDashboard', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<TicketDashboard/>)
  })
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})