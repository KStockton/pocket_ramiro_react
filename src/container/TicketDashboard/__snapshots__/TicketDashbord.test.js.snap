// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TicketDashboard should match the snapshot 1`] = `
<main>
  <section>
    <header>
      <img
        alt="logo"
        src="logo.svg"
      />
      <p>
        Edit 
        <code>
          src/App.js
        </code>
         and save to reload.
      </p>
      <a
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        Learn React
      </a>
    </header>
  </section>
</main>
`;
